lib/teams/server/subscribers/stats-update-subscriber.js

```diff
@@ -15,12 +15,12 @@ class StatsUpdateSubscriber
         let {teamsPlugin, playerSchema} = props;
         let currentTeamId = sc.get(playerSchema, 'currentTeam', '');
         if('' === currentTeamId){
-            Logger.info('Team ID not present.', currentTeamId, playerSchema.currentTeam);
+            // Logger.info('Team ID not present.', currentTeamId, playerSchema.currentTeam);
```

Comment:
uncomment the logger, the info level is ok for this kind of stuff, you can use RELDENS_LOG_LEVEL=8 to avoid seeing these (even when we set LOG LEVEL = 9 by default)

Reply:
yep... my bad this wasn't supposed to be commited.